### PR TITLE
add "qm needs you" blog post

### PR DESCRIPTION
**Pull request by ix0rai**
exactly as in the title!

---
See preview on Cloudflare Pages: https://preview-136.quiltmc-org.pages.dev

### CODE REVIEW

File: src/pages/en/blog/2023-05-19-qm-needs-you.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Quilt Mappings needs you!"
+date: 2023-05-19 12:00:00 -00:00
+index: false
```
Comment (Akarys42) [inline src/pages/en/blog/2023-05-19-qm-needs-you.md:4]: Why? 
```suggestion
```

Comment (ix0rai) [reply src/pages/en/blog/2023-05-19-qm-needs-you.md:4]: good question

**Review by Akarys42 — APPROVED** [review]
Didn't check grammar, but otherwise good

File: src/pages/en/blog/2023-05-19-qm-needs-you.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## How can I help?
+
+The biggest way you can help with QM is by contributing new mappings to the large swaths of the game that are missing names. All you need to know is English and have some basic knowledge of reading code. We have a couple cute documents already written up on how to contribute: check out our [CONTRIBUTING.md](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre2/CONTRIBUTING.md) to find out everything you need to know before mapping! Making new mappings isn’t the only way to help, and if you want to improve our existing names, [our issue tracker](https://github.com/QuiltMC/quilt-mappings/issues) is a great place to find suggestions of names to change. However, be warned that if you change names made by me you run the risk of incurring my wrath (affectionate). Opening Enigma will immediately give you an idea of what needs doing in terms of new mappings, thanks to our handy little status icons next to each class. If a class has a check mark, you can move on, but anything with a yellow dot or red x needs your help! This adorable graph, generated by the Mapping Stats feature in Enigma, can help show you where to start.
```
Comment (ByMartrixx) [inline src/pages/en/blog/2023-05-19-qm-needs-you.md:24]: > All you need to know is English

Something feels off about this. The CONTRIBUTING.md reference should also be updated to the newest branch after Ennui's FCP length adjustments are merged. A link to `good first issue`s would also be nice.

We could also mention here that if someone has a question about how to use Enigma, what to name some method, or any other question, they are welcome to ask in the Discord

File: src/pages/en/blog/2023-05-19-qm-needs-you.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+![A screenshot of a graph generated by Enigma to show unmapped entries, highlighting the world, entity and client packages are problem areas](/assets/img/writing/blog/2023-05-19-qm-needs-you/2023-05-19-mapping-stats.png)
```
Comment (ByMartrixx) [inline src/pages/en/blog/2023-05-19-qm-needs-you.md:26]: The image should be updated to match the latest stats

File: src/pages/en/blog/2023-05-19-qm-needs-you.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+2. Helping out with our deobfuscator, Enigma! Improving Enigma helps us make mappings faster and with fewer mistakes. To help out with Enigma, you can look at *its* [issue tracker](https://github.com/QuiltMC/enigma/issues), or try to break it to help us find more bugs. Enigma is quite a complicated program, and you can always talk to me (rai), or Martrixx on [our Discord](https://discord.quiltmc.org) if you’d like some help understanding it. Enigma is separated into four parts: a backend, a command line interface, a server, and a swing GUI, which means there’s something for every type of Java experience! If the language you speak isn’t English *or* Java, Enigma also has internationalisation support! You can add languages in the [lang](https://github.com/QuiltMC/enigma/tree/master/enigma/src/main/resources/lang) folder, or complete translation for existing languages! Compare against the `en_us` file to find missing translations.
```
Comment (ByMartrixx) [inline src/pages/en/blog/2023-05-19-qm-needs-you.md:31]: Might as well add a notice for our favorite `help wanted` issue, the rare (or sometimes not) QuiltMC/enigma#108

**Review by ByMartrixx — APPROVED** [review]
Great work so far!

File: src/pages/en/blog/2023-05-19-qm-needs-you.md (hunk 2 above)
Comment (ix0rai) [reply src/pages/en/blog/2023-05-19-qm-needs-you.md:24]: I can just remove that bit I think, if you're reading this you already know english

File: src/pages/en/blog/2023-05-19-qm-needs-you.md (hunk 3 above)
Comment (ix0rai) [reply src/pages/en/blog/2023-05-19-qm-needs-you.md:26]: I also need to update the post date, I'll mark it as tomorrow

**Review by obj-obj — APPROVED** [review]
Seems good to me

**Pull request merged**